### ISSUE TITLE
functions starting with `.` not recognized in syntax highlighting

### ISSUE THREAD

**Issue by rsaporta** [randy3k/R-Box#99]
functions starting with `.` not recognized in syntax highlighting

### Example:

```
.test_func <- function(foo, bar="world") {
  print(foo)
}

.test_func(foo="hello", bar="there")
```

**Comment by randy3k** [comment]
I have no issue. Are you using the R Extended syntax?

**Comment by rsaporta** [comment]
Yes, using R Extended. 
Checkout the screenshot

![screen shot 2017-05-18 at 1 08 51 pm](https://cloud.githubusercontent.com/assets/1760870/26214649/2b04b0e4-3bcb-11e7-9190-4ebca72ae8d2.png)

```
.test_func <- function(foo, bar="world") {
  print(foo)
}

## NOTE no highlighting on function name or args
.test_func(foo="hello", bar="there")


test_func2 <- function(foo, bar="world") {
  print(foo)
}

## NOTE yes highlighting on function name and args
test_func2(foo="hello", bar="there")
```

**Comment by randy3k** [comment]
Thank for the report. I confirm it, should be fixed now.

closed by b480aa5
